Makefile.dep

```diff
@@ -21,6 +21,15 @@ ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
   USEMODULE += netdev_default
 endif
 
+ifneq (,$(filter cc2538_rf,$(USEMODULE)))
+  USEMODULE += netdev2_ieee802154
+  ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
+	# XXX: this can be modelled as a dependency for gnrc_netdev_default as soon
+	# as all drivers are ported to netdev2
+    USEMODULE += gnrc_netdev2
+  endif
+endif
+
```

Comment:
Is this needed? I think the Makefile.dep in RIOT's root directory is not the correct place. It is more board- or cpu specific, right?